packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -905,6 +905,10 @@
     "confirm_clear": "Do you really want to clear?",
     "@confirm_clear": {
         "description": "Asking about whether to clear the list or not"
+    },
+    "importance_label": "importance",
+    "@importance_label": {
+        "description": "Used when user selects a food preference"
```

Comment:
Could you add the whole sentence here

Reply:
Hi @M123-dev,
I was unable to do it as `attribute.name`  and `importanceId` are determined at run-time.

Reply:
But some kind of mention that some words are before and after it would be good. It would probably even be better to pass the attribute and the importance to the arb files as inputs so that the translators can choose on the sentence structure. Look at the arb file, you'll see some examples on how to do that. One is the help email body in app_en.arb

Reply:
The second approach looks good to me. I'll pass on the data to `.arb` file directly.